parm/ioda/bufr2ioda/bufr2ioda_adpupa_prepbufr.json

```diff
@@ -0,0 +1,11 @@
+{
+  "data_format"      : "prepbufr",
+  "data_type"        : "prepbufr",
+  "cycle_type"       : "gdas",
+  "cycle_datetime"   : "{{ current_cycle | to_YMDH }}",
+  "dump_directory"   : "{{ DMPDIR }}",
+  "ioda_directory"   : "{{ COM_OBS }}",
+  "subsets"          : [ "ADPUPA" ],
+  "data_provider"    : "U.S. NOAA/NESDIS",
```

Comment:
As far as I know, NESDIS has nothing to do with prepBUFR

Reply:
@CoryMartin-NOAA thanks, removed NESDIS from data provider.